Directory.Build.props

```diff
@@ -13,5 +13,15 @@
     <TargetsWindows Condition="$(RuntimeIdentifier.StartsWith('win'))">true</TargetsWindows>
     <TargetsLinux Condition="$(RuntimeIdentifier.StartsWith('linux'))">true</TargetsLinux>
     <LangVersion>Latest</LangVersion>
+    <!-- We use the compiler toolset that comes from NuGet Packages rather than the SDK built-in.
+    This one sets UseSharedCompilation to false by default. -->
```

Comment:
The comment doesn’t seem to match whith the value you set, should this be false instead?

Reply:
No, actually that is the point, the NuGet package compilers set UseSharedCompilation to false by default, the sdk one sets it to true.

Reply:
Oh I see. I guess the term “this one” is a bit confusing. If you could be a bit more specific there saying we require it to be true but the package one sets the default to false it would be better.

Reply:
Ok, I'll add it.